src/components/Doc/ComponentDoc.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+import { Anchor, Box, Paragraph, Text } from 'grommet';
+import Header from '../Header';
+import RoutedAnchor from '../RoutedAnchor';
+import { Code } from './Code';
+import { DomProps } from './DomProps';
+
+export const ComponentDoc = ({
+  align = 'center',
+  availableAt,
+  children,
+  code,
+  description,
+  intrinsicElement,
+  isA,
```

Comment:
can we add a comment around what `isA` means? it was a little confusing of a variable name for me

Reply:
Yes, I added a comment explaining isA in latest push